website/docs/r/appstream_image_builder.html.markdown

```diff
@@ -56,8 +56,8 @@ The following arguments are optional:
 
 The `access_endpoint` block supports the following arguments:
 
-* `endpoint_type` - (Required) Type of interface endpoint.
-* `vpce_id` - (Optional) Identifier (ID) of the VPC in which the interface endpoint is used.
+* `endpoint_type` - (Required) Type of interface endpoint. Valid values: `STREAMING`
```

Comment:
We've had a lot of times where listing valid values winds up with the options getting out of sync, so I tend towards linking directly to the AWS API docs for this type of thing.

```suggestion
* `endpoint_type` - (Required) Type of interface endpoint. For valid values, refer to the [AWS documentation](https://docs.aws.amazon.com/appstream2/latest/APIReference/API_AccessEndpoint.html)
```

Reply:
Thanks, will follow this convention going forward.